packages/admin/admin/src/stack/breadcrumbs/BreadcrumbsEntry.tsx

```diff
@@ -1,42 +1,89 @@
 import { LevelUp } from "@comet/admin-icons";
-import { IconButton, Link, Typography } from "@mui/material";
-import { WithStyles } from "@mui/styles";
-import clsx from "clsx";
+import { IconButton, Typography } from "@mui/material";
+import Link from "@mui/material/Link";
+import { css, styled, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
 import { BreadcrumbItem } from "../Stack";
 import { BreadcrumbLink } from "./BreadcrumbLink";
-import { styles } from "./StackBreadcrumbs.styles";
 
-interface BreadcrumbsEntryProps {
+const StyledLink: typeof Link = styled(Link, {
+    name: "CometAdminStackBreadcrumbs",
+    slot: "link",
+    overridesResolver(_, styles) {
+        return [styles.link];
+    },
+})(
+    ({ theme }) => css`
+        font-size: 13px;
+        line-height: 14px;
+        font-weight: ${theme.typography.fontWeightMedium};
+        color: ${theme.palette.grey[600]};
+        text-decoration-color: currentColor;
+    `,
+);
+
+const StyledDisabledLink = styled(Typography, {
+    name: "CometAdminStackBreadcrumbs",
+    slot: "disabledLink",
+    overridesResolver(_, styles) {
+        return [styles.disabledLink];
```

Comment:
This should also include the styles of the non-disabled link, as in the previous implementation. 

```suggestion
        return [styles.link, styles.disabledLink];
```